packages/storybook8/stories/Components/VideoGallery/Docs.mdx

```diff
@@ -308,8 +333,33 @@ When `useProps` hook is used to provide the props to VideoGallery from the [stat
 
 The default rendering options can be set for remote video streams as well as the local video stream through VideoGallery props `remoteVideoViewOptions` and `localVideoViewOptions`. To set the remote video streams to fill frame set the scalingMode to 'Crop' like in the code snippet. To set it to fit-to-frame set scalingMode to 'Fit'.
 
-```
-TBD this should be a snippet not bare text. (renderingOptionsDefault)
+```typescript
+const renderingOptionsDefault = `
```

Comment:
Seems like we should remove this line above.

Reply:
we need this so it applies additional typescript formatting